ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -817,154 +814,101 @@ def test_swapaxes(
     )
 
 
-# clip
-@settings(deadline=500)
+@st.composite
+def _tile_helper(draw):
```

Comment:
add comments

Reply:
Code optimisations removed this helper. Added a comment explaining what is happening with the data generation above @given